app/js/controllers/actions/jobs/jobs-list-ctrl.js

```diff
@@ -10,6 +10,32 @@ angular
 		$scope.rdesc = apiDescription.resource(resourceName);
 	});
 
+	$scope.displayDate = function(date) {
```

Comment:
can u move these 3 functions to a utils file so that other controllers can have access to it? L13 - L38